ya_gpt_bot/bot_config/handlers/chat.py

```diff
@@ -117,3 +120,44 @@ async def text_generation_request(  # pylint: disable=too-many-arguments
         await messages_service.save_message(result.message_id, message.message_id, message.chat.id, response, True)
     except ya_exc.GenerationTimeoutError:
         await message.reply(responses.timeout_error)
+
+
+@chat_messages_router.message(Command("digest"))
```

Comment:
When creating a new command, you should add it to `commands.txt` file and help (optionally)